tools/rimage/config/mtl.toml

```diff
@@ -601,9 +601,10 @@ count = 27
 	[[module.entry]]
         name = "RTC_AEC"
```

Comment:
Just for my knowledge: what is RTC_AEC?
Thanks

Reply:
Google webRTC Acoustic Echo Cancellation

Reply:
So, SOF, integrates webrtc library to do AEC?
This is a free library compiled for Xtensa DSP? 
From this link, it seems so: https://webrtc.googlesource.com/src/